GPCS4/SceModules/SceSaveData/sce_savedata_types.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "GPCS4Common.h"
-
+#include <sys/types.h>
```

Comment:
do not include platform specific headers in header files.
better to place all platform related code in `Platform` folder/filter